diff -durN tclgpgme-1.0.orig/Makefile.am tclgpgme-1.0/Makefile.am
--- tclgpgme-1.0.orig/Makefile.am	Fri Aug 30 08:48:32 2002
+++ tclgpgme-1.0/Makefile.am	Sat Jan 11 19:29:47 2003
@@ -17,11 +17,11 @@
 libtclgpgme_la_LIBADD = $(LIB_TCL)
 
 install-exec-hook:
-	$(mkinstalldirs) @PKG_TCL@
+	$(mkinstalldirs) $(DESTDIR)@PKG_TCL@
 	echo	"package ifneeded gpgme"				   \
 		`echo $(VERSION) | awk -F. '{ printf "%s.%s", $$1, $$2 }'` \
-		"[list load $(DESTDIR)$(libdir)/libtclgpgme.so]"	   \
-	    > @PKG_TCL@/pkgIndex.tcl
+		"[list load $(libdir)/libtclgpgme.so]"	   \
+	    > $(DESTDIR)@PKG_TCL@/pkgIndex.tcl
 	sed -e 's%load .libs/libtclgpgme.so%package require gpgme%' \
 	    < test.tcl						    \
-	    > @PKG_TCL@/test.tcl
+	    > $(DESTDIR)@PKG_TCL@/test.tcl
